samples/instrumentation-quickstart/docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
```

Comment:
```suggestion
# Copyright 2024 Google LLC
```

Reply:
These files were technically copied from another repo but Im fine to update them